CHANGES.md

```diff
@@ -1,5 +1,24 @@
 # Changes
 
+## edge-23.4.2
+
+This edge release contains a number of bug fixes.
+
+* CRDs
+  * Fixed Helm uninstall issue for HttpRoute
+
+* CNI
+  * Fixed incompatibility issue with AWS CNI addon in EKS, that was
+    forbidding pods to acquire networking after scaling up nodes.
+  * Added --set flag to install-cni plugin
+
+* Control Plane
+  * Add `trust_dns=error` to default proxy log level 
```

Comment:
This is not really a "control plane" change, it changes a default configuration variable for the proxy. I wouldn't put it under a "control plane" bullet point.